packages/autorest.gotest/src/template/exampleTest.go.njk

```diff
@@ -17,8 +17,8 @@ import (
 {%- set allExamples = [] %}
 {%- for exampleGroup in exampleGroups %}
 {%- for example in exampleGroup.examples %}
-{%- if allExamples.indexOf(example.originalFile)<0 %}
```

Comment:
What's the difference between `originalFile` and `name`?

Reply:
`name` will always be there, but `originalFile` will be empty if `include-x-ms-examples-original-file` set to false.